collect_app/src/main/res/layout/material_full_screen_dialog_toolbar.xml

```diff
@@ -13,11 +13,4 @@
         app:navigationIcon="@drawable/ic_close_black_24dp"
         tools:title="Title" />
 
-    <include
```

Comment:
Why did you remove it?

Reply:
You didn't answer my question but I think you were right you removed the entire view because now it's always gone right and not needed? If that's true please remove it. 

It was just a question because as a reviewer I didn't catch it at first glance, don't hesitate to explain or even argue if you are sure you are right :smiley: just a tip for the future.

Reply:
Hi @grzesiek2010! I removed it because I noticed in the tests that it was not needed now. Thank you for the tip! :D